File: code-examples/java-rs/collation/findOneAndUpdate-default-order-collation.java
/*
   Update the "verified" field to "true" for the first document
   that precedes "Gunter" when sorted lexicographically.
*/
// start findOneAndUpdate default order collation
myColl.findOneAndUpdate(
  Filters.lt("first_name", "Gunter"),
  Updates.set("verified", true)
).subscribe(new Observer<Document>() {

  @Override
  public void onNext(final Document document) {
    System.out.println(document);
  }

  @Override
  public void onError(final Throwable t) {
    System.out.println("Error: " + t.getMessage());
  }

  @Override
  public void onComplete() {
    System.out.println("Operation Completed");
  }
});
// end findOneAndUpdate default order collation

File: code-examples/php/collation/findOneAndUpdate-default-order-collation.php
/*
   Update the "verified" field to "true" for the first document
   that precedes "Gunter" when ordered by using the
   default binary collation order.
*/
// start findOneAndUpdate default order collation
$filter = ['first_name' => ['$lt' => 'Gunter']];
$update = ['$set' => ['verified' => true]];

$options = ['returnDocument' => MongoDB\Operation\FindOneAndUpdate::RETURN_DOCUMENT_AFTER];

$document = $manager->findOneAndUpdate('databaseName.collectionName', $filter, $update, $options);
// end findOneAndUpdate default order collation
